src/Step/ValidatorStep.php

```diff
@@ -79,6 +79,14 @@ public function getViolations()
     }
 
     /**
+     * Use this option to allow extra fields without specific validation constraint set.
+     */
+    public function allowExtraFields()
```

Comment:
Wouldn't it be better to have `addOption('allowExtraFields')`, so other options are also supported?